types/office-js-preview/index.d.ts

```diff
@@ -13215,6 +13278,40 @@ declare namespace Office {
          * @param itemId - The Exchange Web Services (EWS) identifier for an existing calendar appointment.
          */
         displayAppointmentForm(itemId: string): void;
+        /**
+         * Displays an existing calendar appointment.
+         *
+         * The `displayAppointmentFormAsync` method opens an existing calendar appointment in a new window on the desktop or in a dialog box on
+         * mobile devices.
+         *
+         * In Outlook on Mac, you can use this method to display a single appointment that is not part of a recurring series, or the
+         * master appointment of a recurring series, but you cannot display an instance of the series.
+         * This is because in Outlook on Mac, you cannot access the properties (including the item ID) of instances of a recurring series.
```

Comment:
Is it important to explanation the reasons for the limitation? It feels like the developer doesn't need to know this (it may frustrate them if they're not targeting Outlook on Mac).